homeassistant/components/roborock/const.py

```diff
@@ -8,6 +9,44 @@
 CONF_BASE_URL = "base_url"
 CONF_USER_DATA = "user_data"
 
+# domain options
+CONF_INCLUDE_SHARED = "include_shared"
+DEFAULT_INCLUDE_SHARED = True
+
+# Option Flow steps
+SIZES = "sizes"
+DRAWABLES = "drawables"
+MAPS = "maps"
+
+DEFAULT_DRAWABLES = {
+    Drawable.CHARGER: True,
+    Drawable.CLEANED_AREA: False,
+    Drawable.GOTO_PATH: False,
+    Drawable.IGNORED_OBSTACLES: False,
+    Drawable.IGNORED_OBSTACLES_WITH_PHOTO: False,
+    Drawable.MOP_PATH: False,
+    Drawable.NO_CARPET_AREAS: False,
+    Drawable.NO_GO_AREAS: False,
+    Drawable.NO_MOPPING_AREAS: False,
+    Drawable.OBSTACLES: False,
+    Drawable.OBSTACLES_WITH_PHOTO: False,
+    Drawable.PATH: True,
+    Drawable.PREDICTED_PATH: False,
+    Drawable.VACUUM_POSITION: True,
+    Drawable.VIRTUAL_WALLS: False,
+    Drawable.ZONES: False,
+}
+
+DEFAULT_SIZES = {
+    Size.VACUUM_RADIUS: 6,
```

Comment:
Please tell me why a user should change these values.
I currently need help to imagine a reason to change them.

Reply:
It's just stylistically what they want on their image. I figure most users wont ever change it. It's just there to help users fine tune it to look how they would like.

Reply:
For example, I would change some of these values if the map looks too empty or too crowded (depending on scale and house size)

Reply:
Discussed this also internally and it should be removed as it's a rare edge case.